package.json

```diff
@@ -3,75 +3,72 @@
   "private": true,
   "appName": "BCRS Shared Components",
   "scripts": {
-    "storybook": "start-storybook -p 6006",
-    "storybook:build": "build-storybook -o ./docs",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern",
-    "test:unit-silent": "vue-cli-service test:unit --silent",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "storybook": "storybook dev",
+    "build-storybook": "storybook build -o ./docs",
+    "lint": "eslint . --ext js,ts,vue",
+    "lint:nofix": "eslint . --ext js,ts,vue --no-fix",
+    "test:unit": "vitest run",
+    "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
-    "@babel/compat-data": "^7.19.1",
     "@mdi/font": "^5.9.55",
     "axios": "^0.27.2",
-    "core-js": "^3.25.2",
-    "country-list": "^2.2.0",
+    "country-list": "^2.3.0",
     "lodash": "4.17.21",
     "provinces": "^1.11.0",
-    "regenerator-runtime": "^0.13.9",
-    "sbc-common-components": "3.0.1",
+    "regenerator-runtime": "^0.13.11",
+    "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "^2.7.10",
-    "vue-affix": "^0.5.2",
+    "vue": "^2.7.14",
```

Comment:
No, which is why I updated it. Shouldn't matter for the apps that use these components since we only publish the source.

Actually, I will test this, since the components' package files are also published... stand by.

Reply:
Confirmed - the app works fine even if the shared components "depend" on a slightly different version of Vue.

I believe this is because the shared components don't have vue in their own node_modules folder (so no conflict), and because Vue 2.7.10 and 2.7.14 don't have any code differences on our side.

Reply:
Excellent, thanks for exploring this!